### DISCUSSION TITLE
preinstalled copy of this?

### DISCUSSION THREAD

**Discussion by jbmanlol** [voicepaw/so-vits-svc-fork#576] [General]
i need help and i want to install easy please god do it

**Comment by krzysiek546** [comment]
You have all instructions how to install this in [readme](https://github.com/voicepaw/so-vits-svc-fork/blob/1ce4f3d14b29e81fa21ea0fd8bcd6feee14240c5/README.md). Sometimes one click installation don't work properly then you must do manual installation.
